src/general/OptionsMenu.tscn

```diff
@@ -191,10 +192,33 @@ text = "2"
 items = [ "1", null, false, 0, null, "2", null, false, 1, null, "4", null, false, 2, null ]
 selected = 1
 
-[node name="HSeparator" type="HSeparator" parent="TabContainer/Performance"]
+[node name="HBoxContainer3" type="HBoxContainer" parent="TabContainer/Performance"]
 margin_top = 82.0
 margin_right = 1270.0
-margin_bottom = 86.0
+margin_bottom = 106.0
+size_flags_horizontal = 3
+
+[node name="Label" type="Label" parent="TabContainer/Performance/HBoxContainer3"]
+margin_top = 1.0
+margin_right = 617.0
+margin_bottom = 23.0
+hint_tooltip = "This value only applies to new games started after changing this option"
+mouse_filter = 0
+text = "Multisample anti-aliasing (higher values worsen performance): "
+
+[node name="MSAAResolution" type="OptionButton" parent="TabContainer/Performance/HBoxContainer3"]
+margin_left = 621.0
+margin_right = 821.0
+margin_bottom = 24.0
+rect_min_size = Vector2( 200, 0 )
+text = "Disabled"
+items = [ "Disabled", null, false, 0, null, "1", null, false, 1, null, "2", null, false, 2, null, "3", null, false, 3, null, "4", null, false, 4, null ]
```

Comment:
The options should have "2x" "4x" etc. as antialiasing options usually have the x in their values.